packages/ui/src/theme/tokens/components/checkbox.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+  cursor: { value: 'pointer' },
+  alignItems: { value: 'center' },
+  _disabled: {
+    cursor: {
+      value: 'not-allowed',
+    },
+  },
+  button: {
+    position: { value: 'relative' },
+    alignItems: { value: 'center' },
+    justifyContent: { value: 'center' },
+    color: { value: '{colors.white.value}' },
+    before: {
+      width: { value: '100%' },
+      height: { value: '100%' },
+      borderWidth: { value: '{borderWidths.medium.value}' },
+      borderRadius: { value: '20%' },
+      borderStyle: { value: 'solid' },
+      borderColor: { value: '{colors.border.primary.value}' },
+    },
+    _focus: {
+      outlineColor: { value: '{colors.transparent.value}' },
+      outlineStyle: { value: 'solid' },
+      outlineWidth: { value: '{outlineWidths.medium.value}' },
+      outlineOffset: { value: '{outlineOffsets.medium.value}' },
+      borderColor: { value: '{colors.transparent.value}' },
+      boxShadow: { value: '0 0 0 2px {colors.focus.value}' },
+    },
+    _disabled: {
+      borderColor: { value: '{colors.border.disabled.value}' },
+    },
+    _error: {
+      borderColor: { value: '{colors.error.value}' },
+      _focus: {
+        borderColor: { value: '{colors.transparent.value}' },
+        boxShadow: { value: '0 0 0 2px {colors.error.value}' },
+      },
+    },
+  },
+  icon: {
+    borderRadius: { value: '20%' },
+    opacity: { value: '{opacities.0.value}' },
+    transform: { value: 'scale(0)' },
+    transitionProperty: { value: 'all' },
+    transitionDuration: { value: '{time.transition.medium.value}' },
```

Comment:
I think short is probably better. Under 200ms is supposed to feel instantaneous according to UX research, medium is feeling too long.
```suggestion
    transitionDuration: { value: '{time.transition.short.value}' },
```